modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/assets/document-and-media/components/__tests__/DocumentsAndMediaMetricCard.tsx

```diff
@@ -70,17 +70,17 @@ const WrappedComponent = ({empty = false}) => (
 					mockPreferenceReq(THIRTEEN_MONTHS),
 					mockAssetTabsReq({
 						metrics: [
-							DownloadsMetric,
-							PreviewsMetric,
 							CommentsMetric,
+							DownloadsMetric,
+							ImpressionMadeMetric,
 							RatingsMetric
 						],
 						name: NAME,
 						rangeKey: Number(RangeKeyTimeRanges.Last30Days)
 					}),
 					mockAssetMetricReq({
 						empty,
-						metricName: DownloadsMetric.name,
+						metricName: CommentsMetric.name,
```

Comment:
Is that right?

Reply:
@interaminense yes because since I had to sort the const ``metrics`` in `DocumentsAndMediaMetricCard` component I had to sort it in the test too. Otherwise the unit tests breaks